packages/perseus/src/perseus-types.ts

```diff
@@ -127,6 +127,15 @@ export type PerseusRenderer = {
     };
 };
 
+export type Hint = PerseusRenderer & {
+    /**
+     * When `true`, causes the previous hint to be replaced with this hint when
+     * displayed. When `false`, the previous hint remains visible when this one
+     * is displayed.
+     */
+    replace?: boolean;
+};
```

Comment:
The `PerseusRenderer` type already has a `replace` property — should that be removed now?

Reply:
Good catch! Yes, we can remove it now!